### DISCUSSION TITLE
Ingress controller plans

### DISCUSSION THREAD

**Discussion by bgervan** [vitobotta/hetzner-k3s#281] [General]
Hi, 

can you give us some information about your plan about this feature mentioned in the root Readme? How far future is it?

"Note: in a future release it will be possible to configure ingress controllers with host ports, so it will be possible to use an ingress without having to buy a load balancer, but for the time being a load balancer is still required."

**Comment by vitobotta** [comment]
Hi @bgervan, I am working on this in my free time, so sometimes I can add features more quickly, other times it takes longer. I made a release recently and now I am quite busy with my day job and other stuff, so I am not sure when I can make this change. 

There is a workaround if you need this particular change ASAP. Hetzner Cloud allows you to attach more than one firewall to a server, so what you can do now already is create a separate firewall (leaving the one that hetzner-k3s creates alone), and open the ports 80, 83, then attach it to all the worker nodes. This way you can install Nginx or another ingress controller with host port configuration instead of using a load balancer.
